Needed to start the user daemon for newer GDM version.

diff --git a/data/pam-exherbo/gdm-launch-environment.pam b/data/pam-exherbo/gdm-launch-environment.pam
index 3f5d788b1..b4d4d64d9 100644
--- a/data/pam-exherbo/gdm-launch-environment.pam
+++ b/data/pam-exherbo/gdm-launch-environment.pam
@@ -15,3 +15,4 @@ session     required    pam_succeed_if.so audit quiet_success user ingroup gdm
 -session    optional    pam_systemd.so
 -session    optional    pam_elogind.so
 session     optional    pam_permit.so
+-session    optional    pam_openrc.so
